core/app/models/spree/order.rb

```diff
@@ -485,7 +485,7 @@ def can_add_coupon?
     end
 
     def shipped?
-      %w(partial shipped).include?(shipment_state)
+      shipments.shipped.size == shipments.size
```

Comment:
@sebastian-palma I suppose this can be a breaking change as the old method returned true even if it was partially shipped, so maybe rather than using this method it would be better to have a new called `fully_shipped?`